Automattic/wp-calypso
# Reader: Search on Manage, hooking up the UI

First pass at implementing https://github.com/Automattic/wp-calypso/issues/11549.


This PR is not about perfecting the ui, but about getting the skeleton in place for future development.

To test:
- Make sure I haven't broken CombinedCards that depend on fluxPostAdapter. I recommend having wordpress.com stream + this branch side by side and make sure images/metadata haven't changed between this branch + master.
- visually inspect the following/manage url.  make sure that no errors are logged in the console

To Do in a followup pr:
- rename fluxPostAdapter to be fluxReaderAdapter?
